### ISSUE TITLE
The highlight is incorrect when we are mixed using the backquote and double quotation marks.

### ISSUE THREAD

**Issue by ivanzhang1** [jeff-hykin/better-shell-syntax#48]
## The code with a problem is:
```
// code or link to code
#!/bin/sh
if [ ! -d $source_configs_path ]; then
	echo -e `date "+%Y-%m-%d %H:%M:%S" `"[$FUNCNAME] ""The desination path of source_config doesn't exist..."
	mkdir -p $source_configs_path
fi 
```

## It looks like:
** screenshot + theme name preferable **
Theme is default, didn't do any changes once installation is done.
![image](https://user-images.githubusercontent.com/11513419/221455793-193c32c2-29cd-4c09-8888-7f3907f299ca.png)

## It should look like:
** description of ideal **
it is highlighted as below once we uninstalled our extension.
![image](https://user-images.githubusercontent.com/11513419/221456048-acfe0e40-8fe3-4c67-8e7d-8b8f043014ef.png)

**Comment by ivanzhang1** [comment]
vscode 1.73.1
better shell syntax v1.4.1
our code is shell script

**Comment by jeff-hykin** [comment]
So there's two problems, the backticks being one of them, and the other being that command-name is somehow including the first double quote and then not closing itself.

I pushed a fix for the command name, so at least it doesn't ruin everything. The backtick pattern is still one of the old grammars patterns I haven't gotten around to modernizg so I'll have to do that some point in the future.

**Comment by ivanzhang1** [comment]
Thanks for your fixes. Right now, i'm using v1.4.2.

**Comment by jeff-hykin** [comment]
Okay I think this is fully fixed now, v1.8.5
